src/jobflow_remote/cli/flow.py

```diff
@@ -254,9 +245,7 @@ def graph(
         exit_with_error_msg("No data matching the request")
 
     if print_mermaid:
-        from jobflow_remote.jobs.graph import get_mermaid
-
-        print(get_mermaid(flows_info[0]))
+        pass
```

Comment:
This change seems to have removed some functionality. I think several of the rules being applied are not appropriate for a CLI tool

Reply:
good catch. will be reverted in the follow-up PR